libs/schematics/src/ng-add/schema.json

```diff
@@ -0,0 +1,15 @@
+{
+	"$schema": "http://json-schema.org/schema",
+	"$id": "SchematicsNgxNutsAndBoltsNgAddSchema",
+	"title": "ngx-nuts-and-bolts ng add Schema",
+	"type": "object",
+	"properties": {
+		"project": {
```

Comment:
What that this project option do? Does it change anything at all if you change the value of project option in `ng add ... --project=...`?

Reply:
For this specific schematic it isn't necessary nor required. It makes sense for schematics in which you need/can target specific project you want part of code to be generated in.

Reply:
Agreed, please remove this parameter.